apps/mobile/src/screens/adoptions/compose/my-pets-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { AdoptionType } from '@animavita/types';
+import { Box, Heading, View } from 'native-base';
+import { FlatList } from 'react-native';
+
+import { AdoptionCard } from './adoption-card';
+
+import useLocale from '@/hooks/use-locale';
+
+type Adoption = AdoptionType & {
+  id: string;
+};
+
+type MyPetsListProps = {
+  adoptions: Adoption[];
+};
+
+export const MyPetsList = ({ adoptions }: MyPetsListProps) => {
+  if (adoptions.length <= 0) return <EmptyList />;
+
+  return (
+    <FlatList
+      renderItem={({ item }) => <AdoptionCard {...item} />}
+      data={adoptions}
+      keyExtractor={({ id }) => id}
+      ItemSeparatorComponent={() => <View h={4} />}
+    />
```

Comment:
https://github.com/animavita/animavita/assets/19699724/d7fde013-2c5c-49a5-99da-2f2d68c54299

there is a bug when trying to get to the last card in the list